iknow_cache.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
-  s.add_dependency "rails", "~> 5.0.1"
+  s.add_dependency "rails", "~> 5.1.0"
```

Comment:
Yes, this was done before we arrived there. I'll make this one a gem that works and is built against 5.0.x and 5.1.x